package.json

```
@@ -5,7 +5,7 @@
     "dev": "next dev"
   },
   "dependencies": {
-    "next": "13.4.16",
+    "next": "14.0.1",
     "react": "18.2.0",
     "react-dom": "18.2.0",
```

Let's change all of these to `latest` so we don't have to keep manually updating.

```suggestion
    "next": "latest",
    "react": "latest",
    "react-dom": "latest",
```